app/src/main/java/com/example/taskyy/ui/LoginScreen.kt

```diff
@@ -80,6 +80,9 @@ fun LoginScreen(state: LoginState, onEvent: (LoginEvent) -> Unit, navController:
             }
         }
     }
+    if(state.isLoginSuccessful){
+        navController.navigate(Screen.Agenda.route)
+    }
```

Comment:
Same here:
1. Put this in `LaunchedEffect`
2. Pop the login screen from the backstack, so you can't get back to it after logging in